projects/storefrontlib/shared/components/media/media.component.ts

```diff
@@ -69,32 +88,21 @@ export class MediaComponent implements OnChanges {
   @Input() elementType: 'img' | 'picture' = 'img';
```

Comment:
`query:` I think we haven't discussed one more thing. Do we want the `MediaComponent` to use the `img` tag by default at this point? That would mean that `<img>` would be used in all our components except `BannerComponent`, where `elementType` is explicitly set to `picture`.

Reply:
I thought we agreed on this. @Platonn right?

Reply:
As far as I understood, in most places we want to use img - which is product images, which don't need the art direction feature.
The only exception is the banner, for which we want to allow for art direction.

See the [comment](https://github.com/SAP/spartacus/issues/9680#issuecomment-723976093) from @janwidmer:
> So in my opinion, in referral to your [documentation](https://github.com/SAP/spartacus-docs/pull/845/files#diff-784794027f7473410b43510dfaf3c0e4474b8bfea2361ab3760e64e72dc44170R33), if you want to keep the freedom of different ratio's, you would need to adjust the cx-media component to render either img tags with srcset (for e.g. product image with same ratio's) or picture tag with source tags (for e.g. banner with different ratio's per viewport).